app/src/components/submissionPage/SubmissionTextBox.tsx

```diff
@@ -2,49 +2,86 @@ import React from "react";
 import { useState } from "react";
 import Snap from "../../firebase/snap/Snap";
 import submitSnap from "../../firebase/snap/SubmitSnap";
+import { MentionsInput, Mention } from "react-mentions";
```

Comment:
This file is getting a bit big, but I expect it will be refactored in future work by Eli.